### ISSUE TITLE
Add brackets to the filter and other calculations

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#4530]
When using the filter and a single condition, the formula generated has brackets.  When there are 2 (or more?) conditions there are no additional brackets.  This has become very visible recently with the excellent new facilities in the climatic dialogues, e.g. **"(doy_366 >= 1) & doy_366 <= 366"**
This doesn't look good.  I hope it is simple to make it clearer, as: **"(doy_366 >= 1) & (doy_366 <= 366)"**

It will look better if/when we change the names of the columns generated in the Use Date dialogue.  Then it could be **"(doy >= 1) & (doy <= 366)"**, which I think is quite clear - and educational.  

(We tend to use doy_366 a lot and ignore the other one - so doy_366 could become doy and the other could become doy_365!)

**Comment by dannyparsons** [comment]
Yes the brackets should be tidier - either both with brackets or neither.
